### DISCUSSION TITLE
High Availability, IKEv1

### DISCUSSION THREAD

**Discussion by YEONGORI** [strongswan/strongswan#2355] [Q&A] [answered]
Hello,

I have read in the documentation that strongSwan supports active/active redundancy for IKEv2. However, does it support IKEv1 as well? If not, are there any guidelines or recommendations for achieving high availability with IKEv1?

Thank you.

**Comment by tobiasbrunner** [comment, marked as answer]
Technically, yes. But please don't use IKEv1 anymore.
